index.html

```diff
@@ -1,29 +1,30 @@
 <!DOCTYPE html>
 <html>
-<head>
-	<title>Calculator</title>
-	<script src="script.js"></script>
-</head>
 
-<body>
-	<h1>Calculator</h1>
+  <head>
+    <title>Calculator</title>
+    <script src="scripts.js"></script>
+  </head>
 
-	
-        First Number:
-        <input  id="firstNum">
-        
-        +
-       
-        Second Number:
-        <input  id="secondNum">
-        
-        
-        <button onclick="webCalculator()">answer</button>
-        <input id="sum">
-		
+  <body>
+    <h1>Calculator</h1>
+    <form>
```

Comment:
Be careful when you use form tags. I understand you went for readability (which is excellent!), but forms when combined with a submit button refresh the page rather than display your answer. The reason why this occurs is because forms are generally used to connect to a back end server to post some kind of data to a database. So when I attempted to see the result for the calculator, I received an error because the browser was attempting to make a connection with a server that doesn't exist. Once I removed the form tag, it worked perfectly!